### PR TITLE
More tests!

### PR DESCRIPTION
**Pull request by faucomte**
- Random road tests check if the generated road corresponds to the size, branchiness, traffic lights/scenery (on/off) parameters.
- Van crashes tests now test crashes from every possible direction.

### CODE REVIEW

**Comment by faucomte** [comment]
Fixes #841.

**Comment by faucomte** [comment]
**Pull request summary:**
In test_random_road.py, the following tests have been added:
- check that the random road's length is <= to the road size parameter
- check that if branchiness = 0, the random road has no branches.
- check that if branchiness >0, the random road has at least 1 branch.
- check that if traffic lights = no, there are no traffic lights.
- check that if traffic lights = yes, there are traffic lights.
- check that if scenery = no, there is no scenery.
- check that if scenery = yes, there is some scenery.

However, the loopiness and curviness parameters are not fully tested (see issue #842 for more info).

In test_crashes.py, two tests have been added to make sure crashes from all directions were tested:
- check that the van crashes if it goes forward on a right turn.
- check that the van crashes if it goes forward on a left turn.

**Tests runtime:**
Random road: 8 tests run in less than 1 second (unit tests)
Van crashes: 10 tests run in 86 seconds (selenium tests)

**Coverage:**
Total coverage increased by 0.8%.
Coverage in test_random_road.py increased to almost 100%!

**Comment by CelineBoudier** [comment]
Next time please write "Fixes #xxx" in a commit name :)

File: game/tests/test_random_road.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ class RandomRoadTestCase(TestCase):
     def create_test_data(self, num_tiles=None, branchiness=None, loopiness=None, curviness=None,
                          traffic_lights_enabled=False, decor_enabled=True, cows_enabled=False):
         if num_tiles is None:
-            num_tiles = random.randint(3, 30)
+            num_tiles = random.randint(3, 40)
```
Comment (CelineBoudier) [inline game/tests/test_random_road.py]: why did you change the number?
